tembo-cli/Cargo.toml

```diff
@@ -76,6 +76,7 @@ tembo-stacks = "0.4.1"
 itertools = "0.12.1"
 random-string = "1.1.0"
 test-case = "=2.0.0-rc2"
+clap-markdown = { git = "https://github.com/joshuajerin/clap-markdown.git", branch = "0.1.4" }
```

Comment:
Can you move this to a tembo repository before merge

Reply:
done